from PyQt6 import QtCore, QtGui, QtWidgets

class Ui_AttendeeDialog(object):
    def setupUi(self, AttendeeDialog):
        AttendeeDialog.setObjectName("AttendeeDialog")
        AttendeeDialog.resize(500, 350)
        AttendeeDialog.setWindowTitle("Th√™m/S·ª≠a Ng∆∞·ªùi Tham D·ª±")
        
        self.verticalLayout = QtWidgets.QVBoxLayout(AttendeeDialog)
        self.verticalLayout.setObjectName("verticalLayout")
        
        # Form Layout
        self.formLayout = QtWidgets.QFormLayout()
        self.formLayout.setObjectName("formLayout")
        
        # H·ªç T√™n
        self.label = QtWidgets.QLabel("H·ªç T√™n:")
        self.nameInput = QtWidgets.QLineEdit()
        self.nameInput.setPlaceholderText("Nh·∫≠p h·ªç t√™n...")
        self.formLayout.setWidget(0, QtWidgets.QFormLayout.ItemRole.LabelRole, self.label)
        self.formLayout.setWidget(0, QtWidgets.QFormLayout.ItemRole.FieldRole, self.nameInput)
        
        # Email
        self.label_2 = QtWidgets.QLabel("Email:")
        self.emailInput = QtWidgets.QLineEdit()
        self.emailInput.setPlaceholderText("Nh·∫≠p email...")
        self.formLayout.setWidget(1, QtWidgets.QFormLayout.ItemRole.LabelRole, self.label_2)
        self.formLayout.setWidget(1, QtWidgets.QFormLayout.ItemRole.FieldRole, self.emailInput)
        
        # S·ªë ƒêi·ªán Tho·∫°i
        self.label_3 = QtWidgets.QLabel("S·ªë ƒêi·ªán Tho·∫°i:")
        self.phoneInput = QtWidgets.QLineEdit()
        self.phoneInput.setPlaceholderText("Nh·∫≠p s·ªë ƒëi·ªán tho·∫°i...")
        self.formLayout.setWidget(2, QtWidgets.QFormLayout.ItemRole.LabelRole, self.label_3)
        self.formLayout.setWidget(2, QtWidgets.QFormLayout.ItemRole.FieldRole, self.phoneInput)
        
        # T·ªï Ch·ª©c
        self.label_4 = QtWidgets.QLabel("T·ªï Ch·ª©c:")
        self.organizationInput = QtWidgets.QLineEdit()
        self.organizationInput.setPlaceholderText("Nh·∫≠p t·ªï ch·ª©c...")
        self.formLayout.setWidget(3, QtWidgets.QFormLayout.ItemRole.LabelRole, self.label_4)
        self.formLayout.setWidget(3, QtWidgets.QFormLayout.ItemRole.FieldRole, self.organizationInput)
        
        # Ch·ª©c V·ª•
        self.label_5 = QtWidgets.QLabel("Ch·ª©c V·ª•:")
        self.positionInput = QtWidgets.QLineEdit()
        self.positionInput.setPlaceholderText("Nh·∫≠p ch·ª©c v·ª•...")
        self.formLayout.setWidget(4, QtWidgets.QFormLayout.ItemRole.LabelRole, self.label_5)
        self.formLayout.setWidget(4, QtWidgets.QFormLayout.ItemRole.FieldRole, self.positionInput)
        
        self.verticalLayout.addLayout(self.formLayout)
        
        # Spacer
        spacerItem = QtWidgets.QSpacerItem(20, 40, QtWidgets.QSizePolicy.Policy.Minimum, QtWidgets.QSizePolicy.Policy.Expanding)
        self.verticalLayout.addItem(spacerItem)
        
        # Buttons
        self.horizontalLayout = QtWidgets.QHBoxLayout()
        self.horizontalLayout.setObjectName("horizontalLayout")
        
        spacerItem1 = QtWidgets.QSpacerItem(40, 20, QtWidgets.QSizePolicy.Policy.Expanding, QtWidgets.QSizePolicy.Policy.Minimum)
        self.horizontalLayout.addItem(spacerItem1)
        
        self.btnSave = QtWidgets.QPushButton("üíæ L∆∞u")
        self.btnSave.setStyleSheet("padding: 8px 15px;")
        self.horizontalLayout.addWidget(self.btnSave)
        
        self.btnCancel = QtWidgets.QPushButton("‚ùå H·ªßy")
        self.btnCancel.setStyleSheet("padding: 8px 15px;")
        self.horizontalLayout.addWidget(self.btnCancel)
        
        self.verticalLayout.addLayout(self.horizontalLayout)
        
        self.btnCancel.clicked.connect(AttendeeDialog.reject)
        
        QtCore.QMetaObject.connectSlotsByName(AttendeeDialog)
